src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -178,20 +190,23 @@ might produce different results for the recurrences.
 
 .. list-table::
    :class: grid-table
+   :width: 50%
 
    * - .. code-block:: cylc
           :emphasize-lines: 2
 
           [scheduling]
-              initial cycle point = 2000-01-01T00
+              initial cycle point = \
```

Comment:
I don't like this, but it stops the RH column being pushed off the edge of the slide.